doc/input-output-reference/src/overview/group-internal-gains-people-lights-other.tex

```diff
@@ -406,11 +406,11 @@ \subsubsection{Outputs}\label{outputs-017}
 
 \paragraph{People Air Temperature {[}C{]}}\label{people-air-temperature-c}
 
-This output variable represents the zone air temperature based on the Fanger thermal comfort model.~ If there is a ZoneControl:Thermostat:ThermalComfort object specified and the thermal zone is occupied, then the value of ``People Air Temperature'' is determined based on the thermal comfort hat satisfies the thermal comfort setpoint PMV value specified; othwesie, it is set to average zone air temperature.
```

Comment:
Toddler seems likely!

Note that previously there were four volumes I only checked the first, wouldn't hurt to run an English corrector on the rest of the documentation I suspect there are more typos throughout 

Envoyé de mon iPhone

> Le 16 juin 2016 à 19:44, "Michael J. Witte" <notifications@github.com> a écrit :
> 
> In doc/input-output-reference/src/overview/group-internal-gains-people-lights-other.tex:
> 
> > @@ -406,11 +406,11 @@ \subsubsection{Outputs}\label{outputs-017}
> >  
> >  \paragraph{People Air Temperature {[}C{]}}\label{people-air-temperature-c}
> >  
> > -This output variable represents the zone air temperature based on the Fanger thermal comfort model.~ If there is a ZoneControl:Thermostat:ThermalComfort object specified and the thermal zone is occupied, then the value of ``People Air Temperature'' is determined based on the thermal comfort hat satisfies the thermal comfort setpoint PMV value specified; othwesie, it is set to average zone air temperature.
> That's just in a different language. I think it's in Lisp, or toddler.
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub, or mute the thread.
>